homeassistant/components/weatherflow/sensor.py

```diff
@@ -53,25 +53,31 @@ class WeatherFlowSensorEntityDescription(SensorEntityDescription):
 
     event_subscriptions: list[str] = field(default_factory=lambda: [EVENT_OBSERVATION])
     imperial_suggested_unit: None | str = None
+    raw_data_conv_fn: Callable[
+        [Any], datetime | StateType
+    ] = lambda raw_data: raw_data.magnitude
```

Comment:
```suggestion
    ]
```

Reply:
Just to confirm -> you don't want a default value and you'd prefer I set it on each sensor?

Reply:
Yes

Reply:
I'm getting a mypy error:

<img width="937" alt="image" src="https://github.com/home-assistant/core/assets/6491743/674ba68e-25fb-47df-a637-27fb2ce839c2">

> homeassistant/components/weatherflow/sensor.py:53: error: Attributes without a default cannot follow attributes with one  [misc]

Reply:
Is it ok to type ignore that error?

Reply:
I guess you'll pick it up on the re-review...

Reply:
Euh, you need to create a mixin: for example 
https://github.com/home-assistant/core/blob/384adb1c87184a83fb5e9d6cbe84401f5c993ffc/homeassistant/components/youtube/sensor.py#L30

In mixins you can create fields without default value